hash/c64_cass.xml

```diff
@@ -8902,6 +8902,26 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 		</part>
 	</software>
 
+	<software name="labyrintha">
+		<description>Labyrinth (Activision)</description>
```

Comment:
This should use the full title of Labyrinth: The Computer Game.

Reply:
100% agree.  Change implemented.  Thanks for spotting.